src/Middleware/SendRequestDatadogMetric.php

```diff
@@ -43,6 +43,15 @@ public function handle(Request $request, Closure $next)
             unset($tags[$excludeTag]);
         }
 
+        $tagTransformers = config('datadog-laravel-metric.middleware.tag_transformer');
+        // check if $tagTransformers is an array
+        if (is_array($tagTransformers)) {
+            foreach ($tagTransformers as $transClass) {
+                $transformer = app($transClass);
+                $tags = $transformer->transform($tags);
```

Comment:
I think I understand what's going on here but I feel like something missing (but i don't know what 🤔  )

maybe some way to make sure that `$transformer` implements `TagTransformer` ?

Reply:
What is the expectation if the class doesn’t implement the function signature? more proper error message? 🤔

Reply:
like, the `TagTransformer` is in kind of weird position, it is supposed to make sure the Transformer that created is implementing it. But I also can't think the better way to put it.

maybe something like 
```
if !($transformer instanceof TagTransformer) {
    // skip? log error?
}
```

Reply:
I see, actually that makes sense

It's just that the place for explicit interface behavior of dynamic programming like PHP is always awkward, but let me add that validation.

However, if it doesn't satisfy the transformer, I will throw an exception instead of "skip" or "log error", the user of the library should be aware that they need to make the class that satisfy the signature.